### DISCUSSION TITLE
Display kata in-completion rate

### DISCUSSION THREAD

**Discussion by rocco-muscaritolo** [codewars/codewars.com#2859] [Ideas]
Hey, I'm surprised there is no completion statistics available for katas.

I just aborted a 7kyu that required a PhD in mathematics. I'd love to avoid wasting time with kata's that have an abnormally large in-completion rate.

Would also be useful feedback for the kata authors too.

**Comment by hobovsky** [comment]
You mean something like this?

![image](https://user-images.githubusercontent.com/23709795/200966491-588edb01-be0a-4256-bd44-32f20d058087.png)

**Reply by rocco-muscaritolo** [reply]
Yeah, where can I see that?

**Reply by hobovsky** [reply]
Under kata description, at the bottom, in the "Stats" section.

**Reply by rocco-muscaritolo** [reply]
Thanks! Never scrolled past the "similar katas" section.
